example/admin/data_init.go

```diff
@@ -17,6 +17,36 @@ func EmptyDB(db *gorm.DB, tables []string) {
 	}
 }
 
+// DeleteNonEssentialUsers delete all non-admin users but preserve the following three users
+// qor@the-plant.com
+// demo-editor@the-plant.com
+// demo-viewer@the-plant.com
+func DeleteNonEssentialUsers(db *gorm.DB) {
+	preservedUsers := []string{
+		"qor@the-plant.com",
+		"demo-editor@the-plant.com",
+		"demo-viewer@the-plant.com",
+	}
+
+	var adminRoleID int
+	// find the role id for admin role
+	db.Where("name = ?", "Admin").Table("roles").Pluck("id", &adminRoleID)
```

Comment:
It's better to use this const.
https://github.com/qor5/admin/blob/f1f67e0851100ba88cad5d983a30f23e121e72d6/example/models/user.go#L12